cfgmgr/teammgr.cpp

```diff
@@ -553,6 +559,11 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
         conf << ",\"fallback\":true";
     }
 
+    if (lacp_rate == "fast")
```

Comment:
In "teamd.conf", (LACP RUNNER SPECIFIC OPTIONS), fast_rate is defined to be a boolean,

`runner.fast_rate (bool)
	      Option specifies the rate at which our link partner is asked to transmit LACPDU packets. If this is true then packets will  be  sent
	      once per second. Otherwise they will be sent every 30 seconds.`

https://www.unix.com/man-page/centos/5/teamd.conf/
http://manpages.ubuntu.com/manpages/xenial/man5/teamd.conf.5.html

While spawning the teamd process with it's config file, "slow|fast" should be converted to "false|true". Instead why can't we define "fast_rate/lacp_rate" to be a boolean in consistent with "teamd" configuration ?

Reply:
I agree with Arthi, please use fast_rate in the config-DB as well.

Reply:
Answering your question:
* It's easier to read concreate status than to read a boolean flag and convert it to concreate status.
* Boolean flags are harder to maintain (update code base), in case new type need to be added - all code need to be change (boolean flags and conditions using them).
* If it's possible: modules should not depend on low level details. In this example for persion unfarmiliar with teamd it will be easier to find info only about lacp_rate, without dig in teamd.

Thank you for the review, I'll change it to be consistent with links you pointed.

@venkatmahalingam @ArthiSivanantham  could you please share you opinion, do we need to change `lacp_rate` param to `fast_rate` in [config tool too](https://github.com/Azure/sonic-utilities/pull/2036) ?

Reply:
@venkatmahalingam / @ArthiSivanantham IMHO, the arguments provided by @globaltrouble seems to be reasonable. Do we have a strong opinion here? Although, the original approach is compliant with the `teamd.conf` but it is not user friendly.

Reply:
We don't have any strong opinion @nazariig it's just a suggestion as well known teamd uses fast_rate to control the LACP rate and also we can avoid conversion (lacp_rate to fast_rate) in the back-end.